woocommerce/woocommerce
# remove pnpm i from pnpm clean script

### Changes proposed in this Pull Request:





This PR removes `pnpm i` from the `pnpm clean` script.

Closes #47028 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run `pnm clean`
2. In the root of the repo run `find ./ -type d -name node_modules`
3. This should not return results
4. Run `pnpm i`
5. Re-run `find ./ -type d -name node_modules`
6. This should return a long list of folders



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
remove pnpm i from pnpm clean script
</details>
